src/base/trafficGraph.js

```diff
@@ -371,6 +374,10 @@ class TrafficGraph extends EventEmitter {
   setState (state, force, parentState) {
     let updatedState = false;
     if (state && Object.keys(state).length > 0) {
+      // it's important to update entryNode when we're switching views
+      if (state.entryNode) {
+        this.entryNode = state.entryNode;
+      }
```

Comment:
Is there a case where the new state doesn't have an entry node at all? I would we wouldn't need to check if state has an entry node, we should set it regardless.